tiledbcontents/tiledbcontents.py

```diff
@@ -1279,7 +1285,20 @@ def get(self, path, content=True, type=None, format=None):
                 # if model is not None:
                 #     model.
         except Exception as e:
-            raise HTTPError(500, "Error getting {}: {}".format(path_fixed, str(e)))
+            sys.stderr.write(
```

Comment:
I recommend we remove this and instead print a nicer error. Since this is seen by users we want make it for user friendly.

Reply:
Stack trace is removed, now printing a nicer error